src/giddyup.erl

```diff
@@ -27,6 +27,7 @@
 get_suite(Platform) ->
```

Comment:
You could also invert it so the harness knows how to upload artifacts, given a test result ID. Then it could be parallelized across all the hosts in an rtssh setup, or uploaded locally in the runner for a local node.

Just a thought.